File: cfgs/pythoncfgs/COUNT_EVEN_LENGTH_BINARY_SEQUENCES_WITH_SAME_SUM_OF_FIRST_AND_SECOND_HALF_BITS_1.py
Nodes
1;statement;foo_1 = 1;8;
2;statement;foo_1 = 1;9;
3;conditional;foo < foo;10;
4;statement;return foo_1;13;
5;statement;foo_1 = foo_2 * (foo_3 + 1 - foo_4) / foo_5;11;
6;statement;foo_1 += foo_2 * foo_3;12;
7;exit;;
Edges
0;2;
1;3;
2;4;5;
3;7;
4;6;
5;3;
6;

File: cfgs/java_cfgs_foo/COUNT_EVEN_LENGTH_BINARY_SEQUENCES_WITH_SAME_SUM_OF_FIRST_AND_SECOND_HALF_BITS_1.java
Nodes
1;special;;;
2;statement;int nCr = 1;14;
3;statement;int res = 1;14;
4;statement;int r = 1;15;
5;conditional;foo_1 <= foo_2;16;
6;statement;nCr = (foo_1 * ((foo_2 + 1) - foo_3)) / foo_4;18;
7;statement;res += foo_1 * foo_2;19;
8;statement;r++;17;
9;statement;return foo_1;21;
10;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;9;
5;7;
6;8;
7;5;
8;10;
9;
